Sources/ProjectSpec/Dependency.swift

```diff
@@ -31,6 +33,13 @@ public struct Dependency: Equatable {
         self.link = link
         self.implicit = implicit
         self.weakLink = weakLink
+        self.platform = platform
```

Comment:
Please apply `platformDefault` which mentioned above

Reply:
Updated in 07df57d86dfd3172a9e49793ab04a9eb76e7414d.